specification/wse/src/main/scripts/org/kaazing/specification/wse/data/binary.as.escaped.text/echo.escaped.characters/request.rpt

```diff
@@ -0,0 +1,55 @@
+property sequence ${wse:randomInt(100)}
+
+connect http://localhost:8080/path/;e/cte?query
+connected
+
+write method "POST"
+write version "HTTP/1.1"
+write header host
+write header content-length
+write header "X-WebSocket-Version" "wseb-1.0"
+write header "X-Sequence-No" ${wse:asString(sequence)}
+write close
+
+read status "201" /.+/
+read version "HTTP/1.1"
+read header "Content-Type" /text\/plain;charset=(?i)utf-8/
+
+read /(?<upstream>http:\/\/localhost:8080\/path\/.+)/ "\n"
+read /(?<downstream>http:\/\/localhost:8080\/path\/.+)/ "\n"
+read notify CREATED
+read closed
+
+# Downstream
+connect await CREATED
+connect ${downstream}
+connected
+
+write method "GET"
+write version "HTTP/1.1"
+write header host
+write header "X-Sequence-No" ${wse:asString(sequence + 1)}
+write header content-length
+write close
+
+read status "200" /.+/
+read header "Content-Type" "text/plain;charset=windows-1252"
+read header "Connection" "close"
+
+read [0x80 0x04 0x7f 0x30 0x7f 0x72 0x7f 0x6e 0x7f 0x7f]
```

Comment:
["windows-1252 is a superset of IEC_8859-1"](https://en.wikipedia.org/wiki/Windows-1252).  From [IEC_8859-1](https://en.wikipedia.org/wiki/C0_and_C1_control_codes) The popular Windows-1252 character set adds all the missing characters provided by ISO/IEC 8859-15, plus a number of typographic symbols, by replacing the rarely used C1 controls in the range 128 to 159 (hex 80 to 9F).  Thus hex 80 is a valid character in windows-1252
which is padding, or sometimes a euro sign in browsers: http://www.floyd.ch/?p=12.  

Here is a [unicode mapping of 1252](http://www.unicode.org/Public/MAPPINGS/VENDORS/MICSFT/WINDOWS/CP1252.TXT),  0x80 is the euro sign